project/Build.scala

```diff
@@ -49,15 +49,14 @@ object ApplicationBuild extends Build {
   lazy val plugin = Project(appName, base = file("plugin"))
     .settings(baseSettings: _*)
     .settings(
-      resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
-      resolvers += "Typesafe Maven Repository" at "http://repo.typesafe.com/typesafe/maven-releases/",
+      resolvers += "Typesafe Maven Repository" at "https://dl.bintray.com/typesafe/maven-releases/",
       resolvers += "Spy Repository" at "http://files.couchbase.com/maven2",
       resolvers += "Scalaz Bintray Repo"  at "http://dl.bintray.com/scalaz/releases",
-      libraryDependencies += "net.spy" % "spymemcached" % "2.9.0",
+      libraryDependencies += "net.spy" % "spymemcached" % "2.12.0",
       libraryDependencies += "com.typesafe.play" %% "play" % play.core.PlayVersion.current % "provided",
       libraryDependencies += "com.typesafe.play" %% "play-cache" % play.core.PlayVersion.current % "provided",
       libraryDependencies += "com.typesafe.play" %% "play-test" % play.core.PlayVersion.current % "provided,test",
-      libraryDependencies += "org.specs2" %% "specs2" % "2.4.15" % "test",
+      libraryDependencies += "org.specs2" %% "specs2" % "3.6.6" % "test",
```

Comment:
This will break builds for older Play versions since specs2 won't be available for Scala 2.10.x.
I guess that we may be better sticking with 2.4.15 for every build or decide the specs2 version according to the Scala version.